8.gtex-interpret/scripts/nbconverted/0.process-gtex-blood-vae-feature.py

```diff
@@ -154,62 +146,107 @@
 # In[13]:
 
 
+# Output z scores per xCell genesets for all VAE features (5 total)
 file = os.path.join('results', 'gtex_vae_example_interpret_compression.tsv')
-real_df.sort_values(by='z_score').to_csv(file, sep='\t')
+real_df.sort_values(by='z_score').to_csv(file, sep='\t', index=False)
+
+
+# In[14]:
+
+
+# Determine the most distinguishing features between the two models
```

Comment:
What do you mean by distinguisihing?

Reply:
I am talking about the specific genesets that are most differently enriched _in sum_. I will add a comment!